sql/core/src/test/scala/org/apache/spark/sql/sources/BucketedReadSuite.scala

```diff
@@ -1012,4 +1014,182 @@ abstract class BucketedReadSuite extends QueryTest with SQLTestUtils {
       }
     }
   }
+
+  private def checkDisableBucketedScan(
+    query: String,
+    expectedNumBucketedScanEnabled: Int,
```

Comment:
nit: This param name looks a bit confusing and this means the number of bucketed scans when `AUTO_BUCKETED_SCAN_ENABLED` enabled?

Reply:
@maropu - yes. I agree this is bit confusing too, do you have some better name in mind? e.g. `expectedNumScanWithAutoScanEnabled` ?

Reply:
yea, it looks better ;)

Reply:
@maropu - sure, updated.